client/post-editor/style.scss

```diff
@@ -1,3 +1,49 @@
+.dialog.confirmation-dialog {
```

Comment:
The class name for `confirmation-dialog` should be changed to respect CSS class naming guidelines, specifically:

> - The component fragment matches the folder name of the React component it's providing styles to.